source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -1072,9 +1073,12 @@ bool COpenGLDriver::genericDriverInit()
 
     DerivativeMapCreator = new CDerivativeMapCreator(this);
 
+    int32_t dummy;
+    addAndDropMaterialRenderer(new COpenGLSLMaterialRenderer(this, dummy));
+
 	return true;
 }
-
+/*
 class SimpleDummyCallBack : public video::IShaderConstantSetCallBack
```

Comment:
nuke callbacks, add a 256-byte aligned (move around parts of SMaterial as necessary to achieve 0 compiler padding + 256 byte alignment) uint8_t[128] array that will be then used for fetching push-constant data

Reply:
where should this uint8_t[128] live when we get rid of SMaterial?